runtime/src/main/java/org/corfudb/protocols/wireprotocol/IMetadata.java

```diff
@@ -153,6 +153,11 @@ default Long getEpoch() {
                 .get(LogUnitMetadataType.EPOCH)).orElse(Layout.INVALID_EPOCH);
     }
 
+    // TODO(Maithem): replace getGlobalAddress and getEpoch with getToken
+    default Token getToken() {
+        return new Token(getEpoch(), getGlobalAddress());
```

Comment:
Can you please finish TODO?

Reply:
the todo is another story on its own. Maybe you can do it to familiarize yourself with the generator.

Reply:
I did the same for NodeLocator, so not today)